internal/jobs/launch_instance_aws.go

```diff
@@ -140,7 +140,7 @@ func DoEnsurePubkeyOnAWS(ctx context.Context, args *LaunchInstanceAWSTaskArgs) e
 			}
 			ec2Name = pubkey.Name
 		} else {
-			return fmt.Errorf("cannot fetch name of pubkey with fingerpring (%s): %w", fingerprint, err)
+			return fmt.Errorf("cannot fetch name of pubkey with fingerprint (%s): %w", fingerprint, err)
```

Comment:
:bulb: what if we log the fingerprint, but do not include it in the error message itself?
We even have it twice there, because we (me) are also adding it to the parent error: https://github.com/lzap/provisioning-backend/blob/80fe90e3dfb0755e6b55ab408c16d9c75c01bcd0/internal/clients/http/ec2/ec2_client.go#L210

If we log it there, wouldn't that be enough?